_posts/2016-01-25-composer-mailchimp-subscriptions.md

```diff
@@ -0,0 +1,92 @@
+---
+layout: post
+title: "How to Subscribe Users to MailChimp Lists in a Druapl Custom Module"
+date: 2016-01-25
+author: Dan Murphy
+tags: drupal composer module-development mailchimp
+summary: Using composer manager and the MailChimp PHP library we can simply and easily subscribe users to mailing lists without using the MailChimp contributed module.
+
+---
+
+#### Overview
+In my [last blog post](/2015/10/15/composing-with-composer-manager.html), I wrote about the virtues of [Composer Manager](https://www.drupal.org/project/composer_manager) and how it allows modules to depend on PHP libraries managed via Composer. Basically, Composer Manager allows us to easily use PHP libraries that exist outside of the Drupal ecosystem within our own projects.
+
+In this post, I'll show you how we:
+
+- Used the [MailChimp PHP Library](https://packagist.org/packages/mailchimp/mailchimp) in a custom module to quickly and easily subscribe users to a MailChimp mailing list without the overhead of the [Drupal MailChimp contributed module](https://www.drupal.org/project/mailchimp).
+- Managed the MailChimp library dependency via Composer Manager.
+
+#### Custom vs. Conrtib?
+But first, why didn't we just use the MailChimp contributed module? Contributed modules are often a great option and offer many benefits, such as security, maintenance, and flexibility.
```

Comment:
Already noted, just doing it in the diff: Consider taking out "But first"? Does it sound weird starting a new section that way? Maybe it's part of the way you want the blog tone to come across.